.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Install Java (${{ matrix.java }}) and Maven
```

Comment:
Where is your `strategy` defined that contains `java`?

Reply:
Whoops